src/openforms/js/components/admin/form_design/logic/Trigger.js

```diff
@@ -261,8 +267,20 @@ const Trigger = ({ name, logic, onChange }) => {
         }
     }
 
-    const firstOperand = componentType === 'date' ?
-        {date: {var: triggerComponent}} : {var: triggerComponent};
+    let firstOperand;
+    switch (componentType) {
+        case 'date': {
+            firstOperand = {date: {var: triggerComponent}};
+            break;
+        }
+        case 'selectboxes': {
+            firstOperand = {var: `${triggerComponent}.${compareValue}`}
+            compareValue = true;
+            break;
+        }
```

Comment:
Either we should:

- Parse the strings `"false"` and `"true"` from text-input as booleans `false` and `true` or
- Offer a dropdown for type selectboxes component where they can select `True` or `False` which maps to the booleans under the hood (preferred approach)

With the select boxes there's only two states - checked or not checked and you can attach logic to both. By always setting it to checked, you're taking away the option to do something if it's _not_ checked.

Reply:
I thought that since the logic boxes only have 2 states, you can achieve the logic for when the checkbox is unselected by using the `!=` operator